src/Microsoft.Azure.WebJobs.Extensions.OpenApi.Core/Visitors/VisitorCollection.cs

```diff
@@ -49,7 +49,7 @@ public static VisitorCollection CreateInstance()
             var collection = new VisitorCollection();
             collection.Visitors = typeof(IVisitor).Assembly
                                            .GetLoadableTypes()
-                                           .Where(p => p.Name.EndsWith("Visitor") && p.IsClass && !p.IsAbstract)
+                                           .Where(p => p.GetInterface(nameof(IVisitor)) != null && p.IsClass && !p.IsAbstract)
```

Comment:
Instead of using convention ```EndsWith("Visitor")``` which creates conflicts with other assemblies' types (e.g. NewtonsoftJson has some types that end in "Visitor" but are not compatible with the expected logic), check for interface implementations of ```IVisitor```.

Reply:
That's a great catch! Can we by the way use the extension method, `.HasInterface<IVisitor>()`, which is already defined in our `TypeExtensions.cs`, which is basically the same as your change?